server/server.go

```diff
@@ -236,15 +236,15 @@ func (server *HTTPServer) voteOnRule(writer http.ResponseWriter, request *http.R
 	if err != nil {
 		const message = "Unable to get user id"
 		log.Error().Err(err).Msg(message)
-		responses.Send(http.StatusInternalServerError, writer, message)
+		responses.SendError(writer, message)
 		return
 	}
 
 	err = server.Storage.VoteOnRule(clusterID, ruleID, userID, userVote)
 	if err != nil {
-		responses.Send(http.StatusInternalServerError, writer, err.Error())
+		responses.SendInternalServerError(writer, err.Error())
 	} else {
-		responses.Send(http.StatusOK, writer, responses.BuildOkResponse())
+		responses.SendResponse(writer, responses.BuildOkResponse())
```

Comment:
as for me Send with specifying a status code seems to be more readable, but it's a nitpick :)

Reply:
i'm just trying to follow a standard, since I thought we use those utils functions everywhere, it just wasnt nice trying to find all the places where X response is sent. :D